src/BinaryParsers/ElfBinary/Dwarf/DwarfCompileCommandLineInfo.cs

```diff
@@ -18,10 +18,12 @@ public class DwarfCompileCommandLineInfo
         public string CommandLine { get; set; }
         public DwarfLanguage Language { get; set; }
         public string CompileDirectory { get; set; }
+        public bool ParametersIncluded { get; set; } = false;
+        public DwarfCommandLineType DwarfCommandLineType { get; set; } = DwarfCommandLineType.Unknown;
```

Comment:
>DwarfCommandLineType [](http://example.com/codeflow?start=36&length=20)

rename this to CommandLineType, similar to what we used in DwarfLanguage -> Language.

Reply:
agree